src/ray/core_worker/core_worker.cc

```diff
@@ -1812,7 +1799,20 @@ void CoreWorker::HandleWaitForActorOutOfScope(
   if (it == actor_handles_.end()) {
     respond(actor_id);
   } else {
-    RAY_CHECK(actor_out_of_scope_callbacks_.emplace(actor_id, std::move(respond)).second);
```

Comment:
Don't we still need this when gcs actor management is off?

Reply:
actor_out_of_scope_callbacks_ is only used when gcs actor management is turn on.